.github/workflows/sql-release.yml

```diff
@@ -46,7 +47,8 @@ jobs:
         shell: bash
         run: |
           cd sql
-          mvn package -Pnative-lib
+          mvn --batch-mode -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn test 
+          mvn --batch-mode -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn package -Pnative-lib
```

Comment:
Is it necessary to add these flags here or could they be set in the `pom.xml`?

Reply:
moved to the config file

Reply:
Still needs to be removed from this file?

Reply:
removed